src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/ComCaseloadService.kt

```diff
@@ -96,10 +89,8 @@ class ComCaseloadService(
   private fun pairDeliusRecordsWithNomis(managedOffenders: List<DeliusRecord>): List<ManagedCase> {
     val caseloadNomisIds = managedOffenders.filter { offender -> offender.offenderDetail.otherIds.nomsNumber != null }
       .mapNotNull { offender -> offender.offenderDetail.otherIds.nomsNumber }
-    val batchedNomisIds = caseloadNomisIds.chunked(PRISONER_SEARCH_BATCH_SIZE)
-    val batchedNomisRecords = batchedNomisIds.map { batch -> caseloadService.getPrisonersByNumber(batch) }
+    val nomisRecords = caseloadService.getPrisonersByNumber(caseloadNomisIds)
```

Comment:
this method used to pass batches down to two separate calls - one to get prisoners and the other to get court outcomes, both the underlying calls do batching so don't need to worry about batching at this level anymore